core/src/main/scripts/importer/validateData.py

```diff
@@ -4640,6 +4640,7 @@ def processCaseListDirectory(caseListDir, cancerStudyId, logger,
                                 'all_cases_with_mutation_and_cna_and_mrna_data',
                                 'all_cases_with_gsva_data',
                                 'all_cases_with_sv_data',
+                                'all_normal_cases_with_mrna_rnaseq_data',
```

Comment:
We don't need a case list for normal cases. The normal expression data will only be used for reference and additional comparison. Users will not be able to select them for query.